YACS-RCOS/yacs.n
# Update spring 2021 csv

**Issue**

> closes TBA

**Test Procedure**

Show procedure to test functionality with a clear procedure

> 1. Open the tempdeploy
> 2. Use this new spring csv to populate data
> 3. Ensure the functionality works

**Additional Info**

The current Master CSV is empty, and this is to test everything still works
